daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/TestMain.scala

```diff
@@ -90,7 +90,7 @@ object TestMain extends StrictLogging {
             val (apiParameters, cleanup) = if (config.ledgerHost.isEmpty) {
               val sandboxConfig = SandboxConfig.default.copy(
                 port = 0, // Automatically choose a free port.
-                timeProviderType = config.timeProviderType,
+                timeProviderType = Some(config.timeProviderType),
```

Comment:
Not strictly a problem but prefer `Option` over `Some` as a constructor (try `Some(null)` and `Option(null)`).

Reply:
Even when you can guarantee it's not `null`? I'm happy to change it, but we use `Some(…)` all over the place.